app/scripts/app.js

```diff
@@ -48,15 +48,15 @@ angular.module('yourStlCourts').config(function($stateProvider, $urlRouterProvid
       }
     })
     .state('courtSearchInfo', {
-      url: '/courtSearch/info',
+      url: '/courtSearch/info?id',
```

Comment:
Hmmm....So what does the URL produce? /courtSearch/info?id=12345678 ....? The ID should probably be a part of the URL and not a query param. Also, I would suggest renaming "/courtSearch" to somethings like "/courts/{id}"....so in ui-router it would be "url: '/courts/:id'

Later, if we care more about SEO, we should likely use the court name in the URL vs the ID and then do a search by name. Think about keeping URLs simple and embrace simple domain nouns.